### PR TITLE
[2231] Add pgta route

### PR DESCRIPTION
**Pull request by defong**
### Context
PGTA route

### Changes proposed in this pull request
Added PGTA route

### Guidance to review
Works as prototype

### CODE REVIEW

**Comment by defong** [comment]
[[M] Add PGTA route](https://trello.com/c/vJ2U5ejz/2231-m-add-pgta-route)

File: app/lib/training_route_manager.rb [hunk 1]
```diff
@@ -6,50 +6,32 @@ def initialize(trainee)
...
-  def training_route
-    @trainee.training_route
-  end
+  delegate :training_route, to: :trainee
 
-  def feature_enabled?(feature_name)
-    FeatureService.enabled?(feature_name)
+  def enabled?(training_route_enums_key)
+    FeatureService.enabled?("routes.#{training_route_enums_key}") && training_route == TRAINING_ROUTE_ENUMS[training_route_enums_key]
   end
```
Comment (FriedSock) [inline app/lib/training_route_manager.rb:36]: Nice refactor 👍

**Comment by FriedSock** [comment]
@defong I think this isn't being deployed to the review env because of conflicts

**Comment by defong** [comment]
> @defong I think this isn't being deployed to the review env because of conflicts

:face_with_head_bandage: 

Needed to add `deploy` label
image

**Comment by FriedSock** [comment]
Screenshot 2021-07-14 at 11 33 50
Get a 500 error on course details, not sure if this is specific to this route or if seeds need changing? It didn't seem to occur for other routes.

**Review by FriedSock — APPROVED** [review]

**Comment by GeoffJohnson** [comment]
Looks good to me

**Comment by bbelward** [comment]
@defong I get an error message when I select "Check this record." All the sections are complete. 
Screenshot (4)

**Comment by defong** [comment]
> @defong I get an error message when I select "Check this record." All the sections are complete.
> Screenshot (4)

Was discussed elsewhere as this is a wider bug in relation to `diversities` workflow.

**Pull request merged**